__test__/test-website/src/app/[...slug]/page.tsx

```diff
@@ -1,20 +1,41 @@
-import { GraphClient } from '@episerver/cms-sdk';
+import { GraphClient, GraphErrors } from '@episerver/cms-sdk';
 import { OptimizelyComponent } from '@episerver/cms-sdk/react/server';
 import React from 'react';
 
 type Props = {
   params: Promise<{
     slug: string[];
   }>;
+  // Assume that search params are correct:
+  searchParams: Promise<{ variation?: string }>;
 };
 
-export default async function Page({ params }: Props) {
+function handleGraphErrors(err: unknown): never {
```

Comment:
Can't we provide an inbuilt method for logging errors?

Reply:
I think if the SDK throws an error, developers should handle it entirely (including logs). If the SDK doesn't throw an error (for example in case of missing component for content type) then yes, we should log errors.